src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -359,7 +369,7 @@ trait EventFormats {
     )
   }
 
-  def eventToJson(event: MarathonEvent): JsValue = event match {
+  def eventToJson(event: MarathonEvent)(implicit lightweightEvents: LightweightDeploymentPlanFlag): JsValue = event match {
```

Comment:
In line with my previous comment: The implicit should be conversed to an explicit argument